src/Cpanel/PluginActions.php

```diff
@@ -16,33 +16,39 @@ class PluginActions extends AbstractPluginActions
     protected $userConfig;
     protected $composer;
 
-    public static $CONFIG = array(
```

Comment:
I just realized this is CPanel 😄. What is the CPanel min PHP version we support?